CRM/Activity/Form/Activity.php

```diff
@@ -143,7 +143,7 @@ public function getDefaultEntity() {
   public function setFields() {
     // Remove print document activity type
     $unwanted = CRM_Core_OptionGroup::values('activity_type', FALSE, FALSE, FALSE, "AND v.name = 'Print PDF Letter'");
-    $activityTypes = array_diff_key(CRM_Core_PseudoConstant::ActivityType(FALSE), $unwanted);
+    $activityTypes = array_diff_key(CRM_Activity_BAO_Activity::buildOptions('activity_type_id', 'search'), $unwanted);
```

Comment:
These aren't the same. Before it didn't include case activity types and now it does and also all the ones with filter=1.